rust/src/measurements/discrete_gaussian/mod.rs

```diff
@@ -90,6 +90,8 @@ where
 /// | `VectorDomain<AtomDomain<T>>` | `Vec<T>`     | `L2Distance<QI>`        |
 ///
 /// # Arguments
+/// * `input_domain` - Domain of the data type to be privatized. Valid values are `VectorDomain<AtomDomain<T>>` or `AtomDomain<T>`.
+/// * `input_metric` - Metric of the data type to be privatized. Valid values are `AbsoluteDistance<T>` or `L2Distance<T>`.
```

Comment:
I thought you were switching these to use constructor form (i.e. `atom_domain(T)`)?

Reply:
Dropped the second sentence from each, and updated the table. Also brought back the doc comments for the generics. The suppressed generic docs don't appear in Python.